test/formatGeneratedModule-test.ts

```diff
@@ -1,40 +1,5 @@
-import { readFileSync } from "fs";
-import { join } from "path";
-
-import diff from "jest-diff";
-
 import { formatterFactory } from "../src/formatGeneratedModule";
 
-expect.extend({
-  toMatchFile(received, fixturePath) {
-    const actual = readFileSync(fixturePath, "utf8");
-    if (received.trim() === actual.trim()) {
-      return {
-        message: () => `expected ${fixturePath} not to match:\n\n${received}`,
-        pass: true
-      };
-    } else {
-      return {
-        message: () => {
-          const diffString = diff(actual, received, {
-            expand: (this as any).expand
-          });
-          return (
-            this.utils.matcherHint(".toBe") +
-            "\n\n" +
-            `Expected value to be (using ===):\n` +
-            `  ${this.utils.printExpected(actual)}\n` +
-            `Received:\n` +
-            `  ${this.utils.printReceived(received)}` +
-            (diffString ? `\n\nDifference:\n\n${diffString}` : "")
-          );
-        },
-        pass: false
-      };
-    }
-  }
-});
-
```

Comment:
I removed this because I find inline snapshots easier to use. ALso, this caused issues on Windows when files are checked out with non-unix line endings.